Replicator/tests/ReplicatorAPITest.hh

```diff
@@ -241,8 +241,8 @@ public:
 
 
     c4::ref<C4Database> db2;
-    C4Address _address {kDefaultAddress};
-    C4String _remoteDBName {kScratchDBName};
+    C4Address _address = kDefaultAddress;
```

Comment:
Any idea why GCC doesn't like the brace syntax? It and the '=' syntax should be equivalent...